src/App.js

```diff
@@ -3,20 +3,18 @@ import LifeChartHeader from "./components/header/LifeChartHeader"
 import "./i18n.ts";
 import Footer from "./components/footer/Footer";
 import {NavigationContextProvider} from "./context/navigationContext/NavigationContext";
-import {InputContextProvider} from "./context/userinputs/InputContext";
 
 function App() {
   return (
       <>
-          <LifeChartHeader/>
           <NavigationContextProvider>
-              <InputContextProvider>
-                <LifeChart/>
-                <Footer/>
-              </InputContextProvider>
+              <LifeChartHeader/>
```

Comment:
The header can be excluded from the navigation context since it hase nothing to do with it